src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1343,10 +1380,12 @@ rpcClient(std::vector<std::string> const& args,
         if (jvOutput.isMember (jss::error))
         {
             jvOutput[jss::status]  = "error";
-
-            nRet    = jvOutput.isMember (jss::error_code)
-                      ? beast::lexicalCast <int> (jvOutput[jss::error_code].asString ())
-                      : rpcBAD_SYNTAX;
+            if (jvOutput.isMember(jss::error_code))
+                nRet = std::stoi(jvOutput[jss::error_code].asString());
```

Comment:
Great, getting rid of those beastly lexical casts! Another option here is `jvOutput[jss::error_code].asInt()`